test-main.js

```diff
@@ -0,0 +1,24 @@
+var allTestFiles = [];
+var TEST_REGEXP = /(spec|test)\.js$/i;
+
+var pathToModule = function(path) {
+  return path.replace(/^\/base\//, '').replace(/\.js$/, '');
+};
+
+Object.keys(window.__karma__.files).forEach(function(file) {
+  if (TEST_REGEXP.test(file)) {
+    // Normalize paths to RequireJS module names.
+    allTestFiles.push(pathToModule(file));
+  }
+});
+
+require.config({
+  // Karma serves files under /base, which is the basePath from your config file
+  baseUrl: '/base',
+
```

Comment:
I think adding the paths we have defined in common.js here would probably be handy. 

something like 

```
baseUrl: '/base',
paths: {
      //The left side is the module ID
      //the right side is the path relative to baseURL below
      //do not use the file extension
      ko: '/base/build/js/vendor/knockout',
      jquery: '/base/build/js/vendor/jquery',
      bootstrap: '/base/build/js/vendor/bootstrap/bootstrap'
  },
  shim: {
      'bootstrap': {'deps': ['jquery']}
  },

  // dynamically load all test files
  deps: allTestFiles,
```

Oddly, prefixing the paths with /base, was necessary when I did this locally, though the `baseUrl` declaration in common.js seems to work as intended.